src/index.js

```diff
@@ -115,7 +138,11 @@ const getLikedPosts = (timestamp) => {
           for (let i = 0; i < likedPosts.length; i++) {
             const currentPost = likedPosts[i];
             if (Array.isArray(currentPost.photos)) {
-              currentPost.photos.forEach(pushToArray);
+              const filebase = cleanFileName(`${currentPost.blog_name}_${currentPost.timestamp}`);
```

Comment:
What do you think if we create some kind of optional formats for the file names where this can be just one of them? The problem with this change is that at least me, I'm relying a lot on the current names and if I merge this change, I'm going to have a hard time trying to see which files were already there. Do you know what I mean?